app/models/registered.rb

```diff
@@ -0,0 +1,2 @@
+class Registered < OccupationStandard
```

Comment:
I'm thinking these `type` names might get confusing without more context. 

How about `RegisteredStandard`, `UnregisteredStandard`, etc.?

Reply:
Yep, agreed!  I'll make that change.